Dockerfile

```diff
@@ -19,5 +19,9 @@ RUN CGO_ENABLED=0 make install
 
 FROM alpine:latest
 RUN apk add --no-cache ca-certificates
+
+RUN useradd -ms /bin/sh node-drainer
+USER node-drainer
```

Comment:
I would do this after the `COPY`, so the binary is actually owned by root and immutable.

Reply:
Good point!